extensions/parquet-extensions/README.md

```diff
@@ -0,0 +1,4 @@
+Implemented
+1. primitive types support for dimensions and metrics
+2. parquet column filter support i.e. only dimensions, metrics and timestamp will be read
+3. list type support
```

Comment:
can you update the README with information about how someone would use this extension to ingest data from parquet ?